backend/src/routes/emailRouter.js

```diff
@@ -1,13 +1,32 @@
 import express from "express";
 
 const router = express.Router();
+const Email = require("../models/email");
 
 router.get("/", (req, res) => {
-  res.status(200).send("GET: /email - received");
+  Email.find()
+    .then((result) => {
+      res.send(result);
+    })
+    .catch((err) => {
+      // eslint-disable-next-line no-console
+      console.log(err);
+    });
 });
 
 router.post("/", (req, res) => {
-  res.status(201).send("POST: /email - received");
+  const testEmail = new Email({
+    email: "testemail@wdcc.co.nz",
+  });
+  testEmail
+    .save()
+    .then((result) => {
+      res.send(result);
+    })
+    .catch((err) => {
+      // eslint-disable-next-line no-console
+      console.log(err);
```

Comment:
When it comes to logging errors in the console, it is better use "console.error(err)". Both "console.log" and "console.error" shows output in the console, but "console.error" is more effective as it shows red flag that explicitly tells the developers that "this is an error". Have a look at the screenshot below which compares both functions.

![Screen Shot 2021-09-24 at 4 23 41 PM](https://user-images.githubusercontent.com/54789857/134617911-b1051af4-646c-4b8c-8806-505dadfb9e94.png)
.